doc/src/records/dhwsys.md

```diff
@@ -144,22 +155,137 @@ Hourly hot water use (at the point of use).  See further info under wsDayUse.
   ---------- ---------------------- ---------------------- -------------- -----------------
               *name of a DHWDAYUSE*   (no scheduled draws)        No             daily
 
-**wsFaucetWaste=*float***\
-**wsShowerWaste=*float***\
-**wsBathWaste=*float***\
-**wsCWashrWaste=*float***\
-**wsDWashrWaste=*float***
+**wsUnkDrawDurF=*float***
```

Comment:
Let's move Unknown to the end of all the end uses (since it's the exception case).